### ISSUE TITLE
Weird behaviour in Safari on iPhone

### ISSUE THREAD

**Issue by janhenkes** [jedfoster/Readmore.js#201]
We implemented Readmore.js on the following page: https://www.paardenarts.nl/stories/ruby/ (click "Volledig achtergrondverhaal"). We used the following jQuery code to implement it:
```javascript
$('[data-read-more]').readmore({
  speed: 400,
  collapsedHeight: 122,
  moreLink: '<p class="read-more-link"><a href="#">Volledig achtergrondverhaal</a></p>',
  lessLink: '<p><a href="#">Sluiten</a></p>'
});
```

When you open the page on an iPhone and you click "Volledig achtergrondverhaal" and scroll after it opened it closes again, but leaves the close link visible. 

Can you please help me fix this?

**Comment by toongijbels** [comment]
Same behaviour on chrome for ios. (version 58.0.3029.83)

**Comment by jmartsch** [comment]
@toongijbels Just as a note: Chrome uses the rendering engine of Safari on iOS. That´s why you get the same result.

**Comment by emaia** [comment]
Same here

**Comment by dcrec1** [comment]
Same here.

**Comment by robsco** [comment]
Still no resolution to this strange issue?

**Comment by janhenkes** [comment]
@jedfoster Why did you close it? Is it solved?

**Comment by jedfoster** [comment]
Version 3.0 of Readmore.js will drop jQuery entirely, in favor of vanilla JS. I believe this issue will be fixed in 3.0.
